03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>And here offer some explanation that this example is a demonstration of how to modify the system state.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>+1 this would help with prior mentioned confusion about tuners vs updaters</p>